modules/app/src/main/resources/assets/shared-socket/worker/init.ts

```diff
@@ -1,6 +1,5 @@
-import {$isConnected, connect as connectWebSocket} from "../websocket/init";
-import {isMessageWithMetadata} from "./data";
-import {addPort, broadcast, sendTo, sendToId} from "./ports";
+import {$isConnected} from "../websocket/init";
+import {addPort, broadcast, sendTo} from "./ports";
```

Comment:
We're not using double quotes anywhere. Strange that Linter didn't complain.

Reply:
Ah, yeah. Will fix. It's CS, so we have less strict rules here.